pkg/pubsub/provider/provider_test.go

```diff
@@ -0,0 +1,51 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/pubsub/dapr"
+)
+
+func Test_newPubSubSet(t *testing.T) {
+	tests := []struct {
+		name    string
+		pubSubs map[string]InitiateConnection
+		wantKey string
+	}{
+		{
+			name: "only one provider is available",
+			pubSubs: map[string]InitiateConnection{
+				dapr.Name: dapr.NewConnection,
+			},
+			wantKey: dapr.Name,
+		},
```

Comment:
can we have more than 1 dapr provider?

Reply:
Wdym? At a time we can only use one dapr installation in cluster I think. I am not sure if other tools will allow this kind of mechanism.

Reply:
looks like `dapr` and `audit` are hardcoded so we can't have more 1 provider at this time